pymc3/tests/test_dist_math.py

```diff
@@ -16,7 +16,9 @@
 import numpy as np
 import numpy.testing as npt
 import pytest
+import scipy.special as ss
```

Comment:
I would change `as ss` to `as sp`, which I think is a more conventional alias that we use in the rest of the database. I know this is how it was before, but seems more appropriate.